src/EnergyPlus/Fans.cc

```diff
@@ -2386,8 +2397,8 @@ void FanComponent::report(EnergyPlusData &state)
     totalEnergy = totalPower * state.dataHVACGlobal->TimeStepSysSec;
     deltaTemp = outletAirTemp - inletAirTemp;
 
-    if (type == HVAC::FanType::OnOff) {
-        if (airLoopNum > 0) {
+    if (isAFNFan && (airLoopNum > 0)) {
```

Comment:
Is `isAFNFan` another bug fix?

Reply:
I don't remember @mjwitte do you?

Reply:
> Is `isAFNFan` another bug fix?

Not really. Previously `airLoopNum` was being used (abused) only for AFN fans. So, the code assumed that  `airloopNum>0` meant this fan was an AFN fan. When I expanded the use of `airLoopNum` for be set for all fans (to support RMD reports) that was no longer the case, so I added the new `isAFNFan` flag to track that.